### PR TITLE
MSI-2354: Source Region does not set

### PR DESCRIPTION
**Pull request by Stepa4man**
### Fixed Issues
1. magento-engcom/msi#2354: Source Region does not set if only Region Id was selected

### Manual testing scenarios (*)
1. Described in issue

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by swnsma** [comment]
Hi @Stepa4man,

Thank you for your contribution!
Could you please check failed static tests?

**Comment by swnsma** [comment]
@magento run Integration Tests build

**Comment by Stepa4man** [comment]
> Hi @Stepa4man,
> 
> Thank you for your contribution!
> Could you please check failed static tests?

Fixed

File: app/code/Magento/InventoryAdminUi/view/adminhtml/ui_component/inventory_source_form.xml [hunk 1]
```diff
@@ -183,20 +183,22 @@
...
             <formElements>
                 <select>
                     <settings>
-                        <options class="Magento\InventoryAdminUi\Model\OptionSource\RegionSource"/>
                         <filterBy>
                             <field>country_id</field>
                             <target>${ $.provider }:${ $.parentScope }.country_id</target>
                         </filterBy>
                         <customEntry>region</customEntry>
+                        <options class="Magento\Directory\Model\ResourceModel\Region\Collection"/>
```
Comment (swnsma) [inline app/code/Magento/InventoryAdminUi/view/adminhtml/ui_component/inventory_source_form.xml]: Why we need these changes?
**\Magento\InventoryAdminUi\Model\OptionSource\RegionSource** looks like enough to provide list of regions + it has run-cache.

File: app/code/Magento/InventoryAdminUi/view/adminhtml/web/js/form/element/region.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+], function (Region) {
+    'use strict';
+
+    return Region.extend({
+        defaults: {
+            regionScope: 'data.general.region'
+        },
+
+        /**
+         * Set region to customer address form
```
Comment (swnsma) [inline app/code/Magento/InventoryAdminUi/view/adminhtml/web/js/form/element/region.js]: Copy-Paste detected :)

**Review by swnsma — CHANGES_REQUESTED** [review]
Generally - very good.
Please check my notices.

File: app/code/Magento/InventoryAdminUi/view/adminhtml/web/js/form/element/region.js (hunk 2 above)
Comment (Stepa4man) [reply app/code/Magento/InventoryAdminUi/view/adminhtml/web/js/form/element/region.js]: @swnsma So attentive;)

File: app/code/Magento/InventoryAdminUi/view/adminhtml/ui_component/inventory_source_form.xml (hunk 1 above)
Comment (Stepa4man) [reply app/code/Magento/InventoryAdminUi/view/adminhtml/ui_component/inventory_source_form.xml]: Changed back

**Review by swnsma — DISMISSED** [review]

**Comment by Stepa4man** [comment]
@swnsma, please approve this pull request. The issue I told you about earlier will be fixed in separate PR and delivered not to msi, but to magento2 itself.

**Review by ishakhsuvarov — APPROVED** [review]

**Pull request merged**